src/PieChart/PieArcSeries/PieArcLabel.tsx

```diff
@@ -1,23 +1,35 @@
 import React, { PureComponent } from 'react';
 import { motion } from 'framer-motion';
 import { arc } from 'd3-shape';
+import type { ArcData } from '../PieChart';
+import type { ChartInternalDataTypes } from '../../common/data/types';
 import { formatValue } from '../../common/utils/formatting';
 import { findBreakPoint } from './findBreakPoint';
 
-export interface PieArcLabelProps {
-  data: any;
-  centroid: any;
-  format?: (v) => any;
-  fontFill: string;
-  fontSize: number;
+interface OwnProps {
```

Comment:
Based on our convo earlier, I think maybe we should back this out and just do the normal typing.

Reply:
Ok. I'll submit a proposal as a separate issue in future